Cargo.lock

```diff
@@ -2350,7 +2350,8 @@ dependencies = [
 [[package]]
```

Comment:
My rust-analyzer insists on updating the lock file so maybe it's not up to date? Is this update correct?